pkg/kubernetes/diff.go

```diff
@@ -23,7 +23,15 @@ Please upgrade kubectl to at least version 1.18.1.`)
 	// required for separating
 	namespaces, err := k.ctl.Namespaces()
 	if err != nil {
```

Comment:
This would also make the fallback code run on legit errors (server connection failed, etc)

Reply:
In this case, I'm not actually fussed. If there is a problem with the api server, we move on to the next check, and that will fail and report. The user will still get an API server error reported.